Components/Transforms/EulerStackTransform/elxEulerStackTransform.hxx

```diff
@@ -232,11 +232,6 @@ EulerStackTransform<TElastix>::InitializeTransform()
     this->m_Registration->GetAsITKBaseType()->GetFixedImage()->TransformContinuousIndexToPhysicalPoint(
       centerOfRotationIndex, centerOfRotationPoint);
 
-    for (unsigned int k = 0; k < ReducedSpaceDimension; ++k)
-    {
-      redDimCenterOfRotationPoint[k] = redDimCenterOfRotationPoint[k];
-    }
```

Comment:
@stefanklein @whuizinga I think this should have been:

      redDimCenterOfRotationPoint[k] = centerOfRotationPoint[k];

Right?

Reply:
@stefanklein @mstaring This part of the code seems to be there to support specifying CenterOfRotation by index coordinates. Which is deprecated anyway, from elastix version 3.402

https://github.com/SuperElastix/elastix/blob/879edeb8b663406d4c865934614a0a0d2718537a/Components/Transforms/EulerStackTransform/elxEulerStackTransform.h#L65-L67

Would it be OK to simply drop this index-based `CenterOfRotation` parameter by now? For all (seven) transforms that now support _both_ the index-based `CenterOfRotation` _and_ the point-based`CenterOfRotationPoint`? As a follow-up to commit 70dd3b8998d05ff1782eb9a1992c42bb742ddb65 (2 Aug 2006)?

Reply:
- To be resolved by pull request #1306